scripts/globals/items/special_present.lua

```diff
@@ -0,0 +1,28 @@
+-----------------------------------
+-- ID: 5269
+-- Item: Special Present
+-- Starlight Celebration Fame Reward
+-----------------------------------
+require("scripts/globals/status")
+require('scripts/globals/npc_util')
+require("scripts/globals/msg")
+-----------------------------------
+local itemObject = {}
+
+itemObject.onItemCheck = function(target)
+end
+
+itemObject.onItemUse = function(target)
+    local dreamHat = target:hasItem(15178)
+    local dreamHatHQ = target:hasItem(15179)
+
+    if dreamHat and not dreamHatHQ then
```

Comment:
If you drop the Dream Hat you can never get another with this code. Is that intentional?

Reply:
Nope! Will update!

Reply:
Updated to always give a HQ hat if you don't already have one (as expected)!